debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -281,7 +283,7 @@ public static SnapshotMode parse(String value, String defaultValue) {
             .withType(Type.LIST)
             .withWidth(Width.LONG)
             .withImportance(Importance.HIGH)
-            .withValidation(Field::isListOfRegex)
+            .withValidation(Field::isListOfRegex, MongoDbConnectorConfig::validateDatabaseExcludeList)
```

Comment:
This method is called for `DATABASE_WHITELIST` and `DATABASE_INCLUDE_LIST`.  Again this is only aesthetic issue and a different name that express we are doing `#validateDatabaseIncludeExcludeList`.